.github/dependabot.yml

```diff
@@ -11,3 +11,8 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+
+  - package-ecosystem: "docker"
```

Comment:
We have auto-merge dependabot PRs enabled, so you might want to check whether this is going to be included (I think it will be by default). I don't know if that's desirable or not?

Reply:
yes, that's what want, that (combined with continuous deployment) should allow us to easily keep our grafana deployment up to date.